### DISCUSSION TITLE
Toggable Show more  button

### DISCUSSION THREAD

**Discussion by arthurvanl** [scalar/scalar#4323] [Ideas]
![image](https://cdn.discordapp.com/attachments/1135330208627564597/1314507022754119680/image.png?ex=6767cc10&is=67667a90&hm=6554e66dadc41188c0e6b77c46cb374102c503daf34170d9451ed5fc25ff20c0&)

ability to remove the `Show More` button as I would like to defaultly show all routes.

**Comment by arthurvanl** [comment]
bump

**Comment by hanspagel** [comment]
Yess! I’ve created an issue: https://github.com/scalar/scalar/issues/4403
